docs/develop/api-reference/index.mdx

```diff
@@ -196,6 +196,22 @@ puts client.caches.to_a.join(", ")
   cli={`momento cache list`}
 />
 
+### Flush cache
+
+Flushes all data from a cache
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache), simply flushing a cache keeps all settings for this cache and only deletes the data in the cache.
```

Comment:
This sentence uses the word *cache* 5 times. Maybe reword it to something like 
```
While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache), simply flushing a cache keeps all settings and only deletes the data in it.
```

Reply:
Changed wording and pushed new revision.